nexus/src/app/saga.rs

```diff
@@ -63,24 +67,22 @@ impl super::Nexus {
     }
 
     /// Given a saga template and parameters, create a new saga and execute it.
```

Comment:
Nit: The docstring still refers to a "saga template", but IIUC templates are a thing of the past.

Reply:
Good call.  Fixed in 96dba6366f6cb8b47ff10a3a4b0bf0277f9d060e.  I also searched for other references and fixed them.